app/lib/features/chat/widgets/custom_input.dart

```diff
@@ -427,6 +427,7 @@ class __ChatInputState extends ConsumerState<_ChatInput> {
       ),
       builder: (ctx) => AttachmentOptions(
         onTapCamera: () async {
+          Navigator.of(context).pop();
```

Comment:
Not sure if this'd work the same in case of desktop. Hence it wasn't added here. On desktop, attachments open as dialog so this might pop up the chat page. I'd like to be more affirm that it doesn't get broken by attaching screenshot video (Desktop).

Reply:
Won't create any issue on desktop. Here is the video recording of the same.

https://github.com/user-attachments/assets/f9f67ff2-4185-4b0f-b0a4-f5c601ac4349

Reply:
Not a fan of bottom-sheets for desktop UI but I guess that doesn't break it. 👍🏻